tox/session.py

```diff
@@ -298,13 +299,16 @@ def good(self, msg):
         self.logline(msg, green=True)
 
     def warning(self, msg):
-        self.logline("WARNING:" + msg, red=True)
+        if self.verbosity >= -1:
+            self.logline("WARNING:" + msg, red=True)
 
     def error(self, msg):
-        self.logline("ERROR: " + msg, red=True)
+        if self.verbosity >= -1:
```

Comment:
This minus one seems kinda arbitrary - at best should be a constant somewhere - however, even better, maybe the verbosity should be an enum instead, and compare it against that.